### DISCUSSION TITLE
Dissable map icons and labels and change room colors

### DISCUSSION THREAD

**Discussion by adam-kosik** [Tasshack/dreame-vacuum#156] [Q&A]
Hi, is it somehow possible to turn off icons and labels of rendered rooms in map?
And also I would like to change the colors of individual rooms. Is it possible?
Thank you.
![screenshot_2023-06-24 15_03_32](https://github.com/Tasshack/dreame-vacuum/assets/75278121/838b6160-c59b-4f97-91ea-fc1413513218)

**Comment by Tasshack** [comment]
You can disable the lables and icons from the integration configuration settings but you cannot set custom colors to the rooms. Room colors are selected based on an algorithm extracted from the app.
If you really want to set room colors you can try to use the [Valetudo Map card](https://github.com/Tasshack/dreame-vacuum#with-valetudo-map-card).
